service-admin/src/App/src/Middleware/Session/CsrfMiddleware.php

```diff
@@ -28,11 +27,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
 
         if (is_null($csrf)) {
             //  Generate a secret csrf value before proceeding
-            $secret = BigInteger::factory('bcmath')->baseConvert(
-                bin2hex(random_bytes(64)),
-                16,
-                62
-            );
+            $secret = gmp_strval(sprintf('0x%s', bin2hex(random_bytes(64))), 62);
```

Comment:
Should this not be the new token function?

Reply:
OK, now fixed ;)